msteams-platform/TOC.yml

```diff
@@ -269,9 +269,9 @@
       href: bots/calls-and-meetings/registering-calling-bot.md
     - name: Incoming call notifications
       href: bots/calls-and-meetings/call-notifications.md
-    - name: Application-hosted media
+    - name: Requirements of application-hosted media
       href: bots/calls-and-meetings/requirements-considerations-application-hosted-media-bots.md
-    - name: Supported media formats
+    - name: Supported media formats for bots
```

Comment:
Media format for bots

Reply:
Keeping it as: Media format**s** for bots